plugins/baseball-scoreboard/baseball.py

```
@@ -376,6 +471,119 @@ def __init__(
+        try:
+            plays = result_data.get("plays") or []
+            rosters = result_data.get("rosters") or []
+            athlete_names = _build_athlete_name_map(rosters)
+            pitcher, batter = _get_current_pitcher_batter(plays, athlete_names)
+            last_play_code = _get_last_play_code(plays)
+            self._play_by_play_cache[game_id] = {
+                "pitcher": pitcher,
+                "batter": batter,
+                "last_play_code": last_play_code,
+            }
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)